content/v2.0/reference/cli/influxd/_index.md

```diff
@@ -28,18 +28,27 @@ influxd [command]
+| :---------------------       | :---------------------------------------------------------------------------------------------------- | :--------: |
+| `--assets-path`              | Override default assets by serving from a specific directory (developer mode)                         | string     |
+| `--bolt-path`                | Path to boltdb database (default `~/.influxdbv2/influxd.bolt`)                                        | string     |
+| `--e2e-testing`              | Add /debug/flush endpoint to clear stores; used for end-to-end tests (default `false`)                |            |
+| `--engine-path`              | Path to persistent engine files (default `~/.influxdbv2/engine`)                                      | string     |
+| `-h`, `--help`               | Help for `influxd`                                                                                    |            |
+| `--http-bind-address`        | Bind address for the REST HTTP API (default `:9999`)                                                  | string     |
+| `--log-level`                | Supported log levels are debug, info, and error (default `info`)                                      | string     |
+| `--reporting-disabled`       | Disable sending telemetry data to **https:<nolink>//telemetry.influxdata.com**                        |            |
+| `--secret-store`             | Data store for secrets (bolt or vault) (default `bolt`)                                               | string     |
+| `--session-length`           | TTL in minutes for newly created sessions (default `60`)                                              | integer    |
+| `--session-renew-disabled`   | Disables automatically extending session TTL on request                                               |            |
+| `--store`                    | Data store for REST resources (bolt or memory) (default `bolt`)                                       | string     |
+| `--tracing-type`             | Supported tracing types (log or jaeger)                                                               | string     |
+| `--vault-address`            | Address of the Vault server (for example: `https://127.0.0.1:8200/`).                                 | string     |
+| `--vault-ca-cert`            | Path to a PEM-encoded CA certificate file.                                                            | string     |
+| `--vault-ca-path`            | Path to a directory of PEM-encoded CA certificate files.                                              | string     |
+| `--vault-client-cert`        | Path to a PEM-encoded client certificate.                                                             | string     |
+| `--vault-client-key`         | Path to an unencrypted, PEM-encoded private key which corresponds to the matching client certificate. | string     |
```

Comment:
Maybe explicitly state the file extensions in the description? "`.pem` certificate file"?

Also, will want to update this after https://github.com/influxdata/influxdb/pull/15411

Reply:
The file extension doesn't matter. It just needs to be a PEM encoded certificate. InfluxDB/Vault just read the file you pass to them. It can be any filename with any extension.

The SSL config options will be in a separate PR. This PR is just for the Vault-specific configs.